### DISCUSSION TITLE
How can I use this when I'm not in a composable context?

### DISCUSSION THREAD

**Discussion by casaflowa** [adrielcafe/lyricist#29] [Q&A]
How can I use this when I'm not in a composable context? E.g.:

```kotlin
sealed class TabItem(val label: String) {
    object FirstTabItem(label = ???) : TabItem
}
```

In this case I'd usually work with a key. And later in the composable context I'd find the localized string.

Thanks in advance!

**Comment by humblerookie** [comment]
[This PR](https://github.com/adrielcafe/lyricist/pull/40/files) might answer that for you. But before that's merged you could use it in this manner
```
fun getLocaleStrings(): Strings {
  return LyricistStrings[Locale.current.toLanguageTag()] ?: EnStrings
}

```
And use it as `getLocaleStrings().<your-key>`
